src/com_tjvendors/site/tjvendors.php

```diff
@@ -33,3 +34,13 @@
 // Frontend css
 JHtml::_('stylesheet', 'com_tjvendor/tjvendors.css', $options);
 JHtml::stylesheet(JUri::root() . 'media/techjoomla_strapper/vendors/no-more-tables.css', array(), true);
+
+// Load Boostrap
+$params         = ComponentHelper::getParams('com_tjvendor');
```

Comment:
isn't it `com_tjvendors` and not `com_tjvendor` ?

Reply:
Updated